install/helm/open-match/values.yaml

```diff
@@ -135,7 +135,7 @@ openmatch:
       matchmaker_config.yaml:
         logging:
           level: debug
-          format: text
+          format: "{{ternary \"stackdriver\" \"text\" .Values.openmatch.telemetry.stackdriver.enabled}}"
```

Comment:
How to interpret this? how does this get set?

Reply:
This calls the ternary function defined in the gotpl package - http://masterminds.github.io/sprig/defaults.html#ternary. However, helm does not allow templating in the values.yaml file so what we does in v.6 is to quote the templates and pass the strings to the `tpl` function defined in om-configmap.yaml, which makes the `values.yaml` file cumbersome and hard to modify - I have to quote and escape the template to make it works.

This restriction will go away in v0.7 as we introduced the new helm chart.